src/tests/EdFi.DataManagementService.Tests.E2E/Features/SuperclassReferenceValidation.feature.disabled

```diff
@@ -0,0 +1,116 @@
+Feature: SuperclassReferenceValidation of Creation, Update and Deletion of resources
+
+        Background:
+            Given this Local Education Agency exists
+                  | localEducationAgencyId | nameOfInstitution           | localEducationAgencyCategoryDescriptor | educationOrganizationCategoryDescriptor |
+                  | 101                    | Local Education Agency Test | ["Other local education agency"]       | ["Local Education Agency"]              |
+              And this School exists
+                  | schoolId | nameOfInstitution | localEducationAgencyId | educationOrganizationCategoryDescriptor | gradeLevelDescriptor |
+                  | 100      | School Test       | 101                    | ["School"]                              | ["Ninth grade"]      |
+
+        @ignore
+        Scenario: Ensure clients can create a Program that references an existing School
+             When a POST request is made to "ed-fi/programs" with
+                  """
+                  {
+                      "programName": "Program School Test",
+                      "programTypeDescriptor": "uri://ed-fi.org/ProgramTypeDescriptor#Bilingual",
+                      "educationOrganizationReference": {
+                          "educationOrganizationId": 100
+                      }
+                  }
+                  """
+             Then it should respond with 201
+
+        @ignore
+        Scenario: Ensure clients can create a Program that references an existing Local Education Agency
+             When a POST request is made to "ed-fi/programs" with
+                  """
+                  {
+                      "programName": "Program LEA Test",
+                      "programTypeDescriptor": "uri://ed-fi.org/ProgramTypeDescriptor#Bilingual",
+                      "educationOrganizationReference": {
+                          "educationOrganizationId": 101
+                      }
+                  }
+                  """
+             Then it should respond with 201
+
+        @ignore
+        Scenario: Ensure clients cannot create a Program that references a non-existing Education Organization
+            Given educationOrganizationId 102
+             When a POST request is made to "ed-fi/programs" with
+                  """
+                  {
+                      "programName": "Program Non-Existing Test",
+                      "programTypeDescriptor": "uri://ed-fi.org/ProgramTypeDescriptor#Bilingual",
+                      "educationOrganizationReference": {
+                          "educationOrganizationId": 102
+                      }
+                  }
+                  """
+             Then it should respond with 403
+              And the response body is
+                  """
+                  {
+                      "detail": "Access to the requested data could not be authorized.",
```

Comment:
This is not the correct response for the Data Management Service. It occurs in the ODS/API because there is additional security there, that we do not have. Basically, the security is preventing us from even getting to the reference validation problem. The response should be a 409 status code with the same type of error message that you find in `InvalidReferenceValidation.feature`

Reply:
Changed.